### PR TITLE
[TW-964] Breadcrumb redesign

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
To edit the name if Postman elements, you select the name in this release. We're removing the edit icon that appears on hover.

### CODE REVIEW

File: src/pages/docs/collections/using-collections.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ There are several ways to create a new collection:
 
 To customize and configure a new collection, do the following:
 
-1. Select the edit icon <img alt="Edit icon" src="https://assets.postman.com/postman-docs/documentation-edit-icon-v8-10.jpg#icon" width="18px"> to give your new collection a name.
+1. Select the collection name to give your new collection a name.
```
Comment (devanSAGA) [inline src/pages/docs/collections/using-collections.md:70]: Would it make sense to replace Select with Click here?

**Comment by devanSAGA** [comment]
Just FYI, the same rename interaction is also integrated in renaming workspace name - the edit icon is removed and we need to click the workspace name to edit it inside the Workspace Overview page ([postman-app track for reference](https://go.postman-beta.co/?_track=br-feature-AETHER-1841-migrate-to-breadcrumb-v2))

Does this change also requires any change in docs?

**Comment by devanSAGA** [comment]
Do this changes gets deployed somewhere? on a preview track or something? Is there a link that I can compare with existing docs to better understand the changes visually? 

Or do I need to clone the repo and check it locally?

**Review by devanSAGA — CHANGES_REQUESTED** [review]

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 🌟

**Review by devanSAGA — APPROVED** [review]
LGTM 👍

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**